craft_platforms/_architectures.py

```diff
@@ -70,20 +90,58 @@ def to_platform_arch(self) -> str:
         """
         return _ARCH_TRANSLATIONS_DEB_TO_PLATFORM.get(self.value, self.value)
 
+    def to_efi_arch(self) -> str:
+        """Convert this DebianArchitecture to an EFI firmware string.
 
-# architecture translations from the platform syntax to the deb/snap syntax
-_ARCH_TRANSLATIONS_PLATFORM_TO_DEB = {
-    "aarch64": "arm64",
-    "armv7l": "armhf",
-    "i686": "i386",
-    "ppc": "powerpc",
-    "ppc64le": "ppc64el",
-    "x86_64": "amd64",
-}
+        :returns: A string as matched by UKIFY in systemd
+        (see https://github.com/systemd/systemd/blob/main/src/ukify/ukify.py)
+        """
+        return _ARCH_TRANSLATIONS_DEB_TO_EFI.get(self.value, self.value)
 
-# architecture translations from the deb/snap syntax to the platform syntax
+    def to_grub_arch(self) -> str:
+        """Convert this DebianArchitecture to a GRUB boot target.
+
+        :returns: A string suitable for the --target argument to grub-install
+        """
+        return _ARCH_TRANSLATIONS_DEB_TO_GRUB.get(self.value, self.value)
+
+
+# Architecture translation from the deb/snap syntax to (platform, efi, grub) syntaxes
+# - platform: values as returned by uname -m
+# - efi: see EFI_ARCH_MAP in https://github.com/systemd/systemd/blob/main/src/ukify/ukify.py
```

Comment:
Now that we have our own mapping I suggest removing this link. If we really want to keep it for reference I suggest referencing a specific commit (and a specific line?) to make it resistant to future changes/refactoring in systemd.